test/Controller/AuthenticationTest.php

```diff
@@ -83,6 +83,25 @@ public function it_lets_you_log_in_when_the_feature_flag_is_enabled()
         $this->assertContains('Josiah Carberry', $crawler->filter('.login-control')->text());
     }
 
+    /**
+     * @test
+     * @runInSeparateProcess
```

Comment:
Thought `@backupGlobals true` would work...

Reply:
We'll rethink this later with a Python-like context manager